docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -26,8 +26,10 @@ These files are read during boot (and when `systemctl daemon-reload` is run) and
 corresponding regular systemd service unit files. Both system and user systemd units are supported.
 
 The Podman generator reads the search paths above and reads files with the extensions `.container`
-`.volume` and `*.kube`, and for each file generates a similarly named `.service` file. These units
-can be started and managed with systemctl like any other systemd service.
+`.volume` and `*.kube`, and for each file generates a similarly named `.service` file. Be aware that
+existing user and system services are replaced if they have the same name. The generated unit files can
```

Comment:
This is not quite true, see https://github.com/containers/podman/issues/18275#issuecomment-1519685487.
The current `normal-dir` only causes us to overwrite "vendor" services (`/usr/...`) but the ones from `/etc/...`

Reply:
Really nice catch.  I've updated the text. PTanotherL